.eslintrc.json

```diff
@@ -36,6 +36,8 @@
     "react/display-name": "off",
     "jsx-a11y/label-has-for": "off",
     "jsx-a11y/label-has-associated-control": "off",
-    "jest/no-disabled-tests": "off"
+    "jest/no-disabled-tests": "off",
```

Comment:
Why do we want to allow disabled tests? Couldn't we just comment them out if they're not currently working?

Reply:
This is a straight copy from react-components repo

Reply:
Ha we should probably fix that then

Reply:
Ok i'll remove it from here then